tasks/base/SLES-12/install_docker.yml

```diff
@@ -12,35 +12,6 @@
   delay: 30
   until: remove_packages is success
 
-- name: Add docker repository
```

Comment:
SP 5 comes already with a suitable repo for docker 19, so no need to install an other one